test/screenshot/infra/commands/travis.sh

```diff
@@ -65,12 +67,13 @@ function install_and_authorize_gcloud_sdk() {
     # The gcloud installer runs `tar -C "$install_dir" -zxvf "$download_dst"`, which generates a lot of noisy output.
     # Filter out all lines from `tar`.
     /tmp/gcp-sdk.bash | grep -v -E '^google-cloud-sdk/'
+
+    gcloud components install gsutil
```

Comment:
What does moving these do?

Reply:
It prevents the script from failing with this error when running in a Travis VM:

```
ERROR: (gcloud.components.install) You cannot perform this action because this Cloud SDK installation is managed by an external package manager.
Please consider using a separate installation of the Cloud SDK created through the default mechanism described at: https://cloud.google.com/sdk/

ERROR: (gcloud.components.update) You cannot perform this action because this Cloud SDK installation is managed by an external package manager.
Please consider using a separate installation of the Cloud SDK created through the default mechanism described at: https://cloud.google.com/sdk/
```

See my previous comment for more info: https://github.com/material-components/material-components-web/pull/3931#issuecomment-430491586

Reply:
Oh derp I didn't realize we moved it into an else, whereas it was unconditional before.